tests/test_input_damages.py

```diff
@@ -0,0 +1,1287 @@
+            "gdp": (["ssp", "region", "model", "year"], np.full((2, 2, 2, 2), 1)),
+            "pop": (["ssp", "region", "model", "year"], np.full((2, 2, 2, 2), 1)),
+        },
+        coords={
+            "ssp": (["ssp"], ["SSP2", "SSP3"]),
+            "region": (["region"], ["ZWE.test_region", "USA.test_region"]),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "year": (["year"], [2010, 2099]),
+        },
+    )
+
+    d = os.path.join(tmp_path, "econvars_for_test", "econvars_for_test.zarr")
+    econvars.to_zarr(d)
+    econvars_for_test = EconVars(path_econ=d)
+
+    return econvars_for_test
+
+
+def test_calculate_labor_impacts(
+    tmp_path,
+    labor_in_val_fixture,
+    labor_in_histclim_fixture,
+    file_prefix="uninteracted_main_model",
+    variable="rebased",
+    val_type="wage-levels",
+):
+    """
+    Test that calculate_labor_impacts correctly calculates the impacts for labor results
+    """
+    ds_out_expected = xr.Dataset(
+        {
+            "histclim_rebased": (["year", "region"], np.full((2, 2), 2)),
+            "delta_rebased": (["year", "region"], np.full((2, 2), 0)),
+        },
+        coords={
+            "year": (["year"], [2010, 2099]),
+            "region": (["region"], ["ZWE.test_region", "USA.test_region"]),
+        },
+    )
+
+    d = os.path.join(
+        tmp_path, "labor_in", "batch6", "rcp45", "ACCESS1-0", "high", "SSP3"
+    )
+
+    ds_out_actual = calculate_labor_impacts(d, file_prefix, variable, val_type)
+    xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+
+@pytest.mark.parametrize("out_format", ["return", "save"])
+def test_concatenate_labor_damages(
+    tmp_path,
+    econvars_fixture,
+    labor_in_val_fixture,
+    labor_in_histclim_fixture,
+    out_format,
+):
+    """
+    Test that concatenate_labor_damages correctly concatenates damages across batches
+    """
+    ds_out_expected = xr.Dataset(
+        {
+            "histclim_rebased": (
+                ["ssp", "rcp", "model", "gcm", "batch", "year", "region"],
+                np.float32(np.full((1, 2, 2, 2, 1, 2, 2), 2 * -1 * 1.273526)),
+            ),
+            "delta_rebased": (
+                ["ssp", "rcp", "model", "gcm", "batch", "year", "region"],
+                np.float32(np.full((1, 2, 2, 2, 1, 2, 2), 0)),
+            ),
+        },
+        coords={
+            "year": (["year"], [2010, 2099]),
+            "region": (["region"], ["ZWE.test_region", "USA.test_region"]),
+            "ssp": (["ssp"], ["SSP3"]),
+            "rcp": (["rcp"], ["rcp45", "rcp85"]),
+            "gcm": (["gcm"], ["ACCESS1-0", "GFDL-CM3"]),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "batch": (["batch"], ["batch9"]),
+        },
+    )
+
+    ds_out_actual = concatenate_labor_damages(
+        input_path=os.path.join(tmp_path, "labor_in"),
+        save_path=tmp_path,
+        ec_cls=econvars_fixture,
+    )
+
+    if out_format == "return":
```

Comment:
why the two `out_format` options?

Reply:
This is because `concatenate_labor_damages` func simultaneously saves output for each batch and returns the output for the last batch being processed:
https://github.com/ClimateImpactLab/dscim/blob/1aaa8e286ad7e269c5110ee46634e0a98cced6c7/src/dscim/preprocessing/input_damages.py#L223-L233

A `return()` statement is definitely unnecessary here so we might want to remove it.

Reply:
Sounds good. I think it's fine to leave it. Thanks!